engine/apps/slack/views.py

```diff
@@ -285,55 +301,73 @@ def post(self, request):
                 # Open pop-up to inform user why OnCall bot doesn't work if any action was triggered
                 self._open_warning_window_if_needed(payload, slack_team_identity, warning_text)
                 return Response(status=200)
+        elif organization is None and payload_type_is_block_actions:
```

Comment:
`and payload_type_is_block_actions` is the main change of this PR. 

From what I can see from the logs,`Organization matching query does not exist.` exceptions originating from this endpoint only seem to be happening when `payload_type == 'block_actions'`